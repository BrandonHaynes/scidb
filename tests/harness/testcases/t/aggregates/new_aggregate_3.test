--setup

CREATE ARRAY expr1 <a:int64, b:int64>[x=1:5,2,0, y=1:5,2,0]

load(expr1, '${TEST_DATA_DIR}/expr1.txt')

--test
aggregate(expr1, sum(a), sum(b), y)

--cleanup
remove(expr1)
