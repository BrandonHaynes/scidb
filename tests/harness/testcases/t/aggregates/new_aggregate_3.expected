Query was executed successfully

{x,y} a,b
{1,1} 2,4
{1,2} 6,8
{2,1} 22,24
{2,2} 26,28
{1,3} 10,12
{1,4} 14,16
{2,3} 30,32
{2,4} 34,36
{1,5} 18,20
{2,5} 38,40
{3,1} 42,44
{3,2} 46,48
{4,1} 62,64
{4,2} 66,68
{3,3} 50,52
{3,4} 54,56
{4,3} 70,72
{4,4} 74,76
{3,5} 58,60
{4,5} 78,80
{5,1} 82,84
{5,2} 86,88
{5,3} 90,92
{5,4} 94,96
{5,5} 98,100

{y} a_sum,b_sum
{1} 210,220
{2} 230,240
{3} 250,260
{4} 270,280
{5} 290,300

Query was executed successfully

